src/snowflake/cli/plugins/git/commands.py

```diff
@@ -42,6 +42,24 @@ def _repo_path_argument_callback(path):
 )
 
 
+def like_option(help_example: str):
+    return typer.Option(
+        "%%",
+        "--like",
+        "-l",
+        help=f"SQL LIKE pattern for filtering objects by name. For example, {help_example}",
+    )
```

Comment:
Why not use `snowflake.cli.api.commands.flags.LikeOption`?

Reply:
Refactor with https://github.com/snowflakedb/snowflake-cli/pull/889

Reply:
[merged]